src/canopy_calcs.F90

```diff
@@ -216,116 +227,154 @@ SUBROUTINE canopy_calcs
                                     !ISOP
                                     call canopy_bio(zk, fafraczInt, hcmref, &
                                         lairef, fsun, ppfd_sun, ppfd_shade, tleaf_sun, tleaf_shade, &
-                                        tleaf_ave, &
+                                        tleaf_ave, tmp2mref, &
                                         lu_opt, vtyperef, modres, bio_cce, biovert_opt, co2_opt, co2_set, &
+                                        leafage_opt, pastlai, currentlai, tsteplai, tabovecanopy, &
```

Comment:
@quaz115 Why is `tabovecanopy` still input here?  Aren't  we just using `tmp2mref`?